auth/auth.css

```diff
@@ -0,0 +1,128 @@
+*{
+    font-family: 'Ubuntu', sans-serif;
+}
+
+:root{
+    --color-body:#f5f8fc;
+    --color-card:white;
+    --color-lightblue:#a7d4f7;
+    --color-grey:#d2d7dd;
+    --color-blue:#5959ec;
+    --color-btn-hover:#383CC1;
+}
+
+body{
+    background-color: var(--color-body);
+    line-height: 1.5;
+    font-weight: 500;
+    font-size: large;
+}
+
+.login-card{
+    margin: 5rem auto;
+    min-width:280px;
+    width:310px;
+    background-color: var(--color-card);
+    padding:2rem;
+    border-radius: 4px;
+    height: fit-content; 
+    box-shadow: 0 4px 4px 0 rgb(0 0 0 / 20%);
+}
+
+.flex-col{
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+}
+
+.flex-row{
+    display: flex;
+    gap: 1rem;
+    align-items: center; 
+    justify-content: center;
+}
+.flex-center{
+    flex-direction: row;
+    gap: 2px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.flex-left{
+    display: flex;
+    justify-content: flex-start;
+}
```

Comment:
you can make a separate class for common css properties or just use the properties made in component library

Reply:
yeah i ll make use of it